crates/bindings-csharp/Runtime/Internal/FFI.cs

```diff
@@ -221,4 +223,10 @@ public static partial CheckedStatus _bytes_sink_write(
         ReadOnlySpan<byte> buffer,
         ref uint buffer_len
     );
+
+    [LibraryImport(StdbNamespace)]
+    public static partial uint _console_timer_start([In] byte[] name, uint name_len);
```

Comment:
Please add strongly-typed id wrapper struct like we do for other IDs. It serves both as a documentation and makes it harder to pass the wrong integer into the wrong API, but in marshalling it behaves exactly the same way.

Reply:
Done, although it didn't like PInvoke method having a custom managed return type, so I had to add a custom Marshaller. Doesn't look like we had `TableId` or any other managed types as method returns previously (aside from `CheckedStatus`, which also has a custom Marshaller), so it may be a limitation of C# / WASM/ whatever

Reply:
Ugh yeah we used to have to do that for `Buffer`, we just don't return it by value anymore. In theory this should be among things the new FFI in .NET 9 fixes, fingers crossed.